gnocchi/tests/base.py

```diff
@@ -268,9 +269,15 @@ class TestCase(BaseTestCase):
     @classmethod
     def setUpClass(self):
         super(TestCase, self).setUpClass()
-        self.conf = service.prepare_service([],
-                                            default_config_files=[])
-        if not os.getenv("GNOCCHI_TEST_DEBUG"):
+
+        debug = "GNOCCHI_TEST_DEBUG" in os.environ
+
+        self.conf = service.prepare_service(
+            [],
+            default_config_files=[],
+            logging_level=logging.DEBUG if debug else logging.NOTSET)
```

Comment:
what's logging.NOTSET? that resolves to 0 which is less than logging.DEBUG so it would print everything also?